consensus/ibft/snapshot.go

```diff
@@ -366,21 +366,29 @@ func newSnapshotStore() *snapshotStore {
 }
 
 // loadFromPath loads a saved snapshot store from the specified file system path
-func (s *snapshotStore) loadFromPath(path string) error {
+func (s *snapshotStore) loadFromPath(i *Ibft) error {
 	// Load metadata
 	var meta *snapshotMetadata
-	if err := readDataStore(filepath.Join(path, "metadata"), &meta); err != nil {
-		return err
-	}
+	if err := readDataStore(filepath.Join(i.config.Path, "metadata"), &meta); err != nil {
+		// if we can't read file metadata file delete it and sync again
+		// and logout that we've had some problems
+		i.logger.Error("could not read metadata file","err",err.Error())
```

Comment:
Please change the log output from `metadata` to `snapshot store`, in the entire method

Reply:
Is there any chance for us to see the actual error and compare it before wiping out the snapshot store file?

Reply:
FIxed in commit b0bf7cd004339996424fc7301b460a33873b367f
The actual error will be outputted to console logs, something similar to this:
```
 2022-01-17T14:27:09.155Z [ERROR] polygon.consensus.ibft: Could not read snapshot data: err="unexpected end of JSON input"
```

Reply:
Got it, makes sense.
The file is empty so there's nothing to unmarshal for the json module